### ISSUE TITLE
about the Copy_Paste

### ISSUE THREAD

**Issue by 12moli** [ultralytics/yolov5#10014]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

The number of samples in each class of my dataset is very uneven, as follows：
![image](https://user-images.githubusercontent.com/102312535/199495893-f5f3edd4-6fc5-4a3e-a1a8-1dc0b40f3da6.png)
I want to expand a category with less than 1000 samples to 1000. I am using Copy_paste for data enhancement, should I do the Copy_paste operation offline or adjust the hyperparameters in yolov5 for the online Copy_paste operation?

**Comment by glenn-jocher** [comment]
YOLOv5 copy-paste is only used for datasets providing segmentation labels, so if you only have box labels I'd do it offline.

**Comment by 12moli** [comment]
> YOLOv5 copy-paste is only used for datasets providing segmentation labels, so if you only have box labels I'd do it offline.

Thank you for your reply! I recently modified the backbone of yolov5 and after training, the loss in the train section is normal, but the loss in the val section is abnormal, the loss is nan at the beginning. Map, Recall, and Precision is all 0. Is this because of the error caused by my modified network model? The results are as follows.
![1](https://user-images.githubusercontent.com/102312535/200158925-fcc7ee88-20ce-486e-a31e-37191f2e14a0.png)

**Comment by glenn-jocher** [comment]
👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃
